Car_maker_task/src/main/java/com/epam/cars/H2CarRepository.java

```
@@ -0,0 +1,159 @@
+                    }
+                }
+
+                Car car = new Car(makerRep.getMaker(id_maker), model, year, color);
+                car.setId(id_car);
+                cars.put(id_car, car);
+            }
+            rs.close();
+
+        } catch (SQLException sqlEx) {
+            // sqlEx.printStackTrace();
+        } finally {
+            try {
+                con.close();
+            } catch (SQLException se) {
+            }
+            try {
+                stmt.close();
+            } catch (SQLException se) {
+            }
+            try {
+                rs.close();
+            } catch (SQLException se) {
+            }
+        }
+
+        return new ArrayList(cars.values());
+    }
+
+    @Override
+    public void saveCar(Car car) {
+        try {
+            con = DriverManager.getConnection(url, user, password);
+            pstmt = con.prepareStatement("Insert into CAR Values(?,?,?,?,?");
+            pstmt.setLong(1, ++lastCarId);
+            pstmt.setString(2, car.getModel());
+            pstmt.setString(3, car.getColor());
+            pstmt.setInt(4, car.getYear());
+            pstmt.setLong(5, car.getMaker().getId());
+            pstmt.executeQuery();
+
+        } catch (SQLException ex) {
+        } finally {
+            try {
+                con.close();
+            } catch (SQLException se) {
+            }
+        }
+    }
+
+    @Override
+    public void updateCar(Long id, Car car
+    ) {
+        try {
+            con = DriverManager.getConnection(url, user, password);
+            pstmt = con.prepareStatement("Update CAR set MODEL=? COLOR=? YEAR=? ID_MAKER=? WHERE ID_CAR=?");
+            pstmt.setString(1, car.getModel());
+            pstmt.setString(2, car.getColor());
+            pstmt.setInt(3, car.getYear());
+            pstmt.setLong(4, car.getMaker().getId());
+            pstmt.setLong(5, id);
+            pstmt.executeQuery();
+
+        } catch (SQLException ex) {
```

Why did you swallow exception??? Are you not interested if your update successful or it fails with error?